diagnostics-app/src/app/shared/header/header.component.ts

```diff
@@ -18,6 +18,7 @@ import {APP_NAME} from 'common/config';
   styleUrls: ['./header.component.css'],
 })
 export class HeaderComponent {
+  @Input() showToggleSideNavButton = true; // Default to showing the toggle button
```

Comment:
Would it be better to put this config variable to `diagnostics-app/src/common/config/common.ts`?

Reply:
I don't think so for this case.

I've add it as an input here so that I can have different behavior in my support assist app and RMA app.

For support assist, I would like to be able to open and close the side menu. This variable therefore should be true.

For RMA, I want the side bar to always show, as that is where the testlist is, like in factory toolkit. The variable should be set to false in this case.

Reply:
Got it. I misunderstood that you will build either support assist or RMA app at a time.

Thanks for the explanation.